<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="restWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>911</width>
     <height>481</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="publish">
    <attribute name="title">
     <string>Publish</string>
    </attribute>
    <widget class="QLabel" name="frontTempLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Front Temp</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="rearTempLabel">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>280</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Rear Temp</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lshLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>280</y>
       <width>41</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Left Seat Heat</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="rshLabal">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>280</y>
       <width>41</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Right Seat Heat</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDial" name="fanSpeed">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>70</y>
       <width>131</width>
       <height>131</height>
      </rect>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLCDNumber" name="ambientTemp">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>200</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="isACActive">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>270</y>
       <width>50</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>A/C</string>
     </property>
    </widget>
    <widget class="QScrollBar" name="isRecirculationActive">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>270</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>121</width>
       <height>181</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QSlider" name="frontTemperature">
        <property name="minimum">
         <number>15</number>
        </property>
        <property name="maximum">
         <number>30</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollBar" name="leftSeatTemp">
        <property name="maximum">
         <number>3</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="invertedAppearance">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>100</y>
       <width>121</width>
       <height>181</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QScrollBar" name="rightSeatTemp">
        <property name="maximum">
         <number>3</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="invertedAppearance">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="rearTemperature">
        <property name="minimum">
         <number>15</number>
        </property>
        <property name="maximum">
         <number>30</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>290</y>
       <width>90</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Recirculation</string>
     </property>
    </widget>
    <widget class="QPushButton" name="publishButton">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>380</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Publish</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="retrieve">
    <attribute name="title">
     <string>Retrieve</string>
    </attribute>
    <widget class="QTabWidget" name="retriveTab">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>881</width>
       <height>431</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tableTab">
      <attribute name="title">
       <string>Table</string>
      </attribute>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>851</width>
         <height>331</height>
        </rect>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Temperature(Front)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Temperature(Rear)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Heat (Left Seat)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Heat (Right Seat)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Fan Speed</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Temperature (Ambient)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>A/C Active</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Recirculation Active</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Created at</string>
        </property>
       </column>
      </widget>
      <widget class="QPushButton" name="tablePopulate">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>360</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Retrieve</string>
       </property>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>350</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Results</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="numResults">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>15</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="graphTab">
      <attribute name="title">
       <string>Graph</string>
      </attribute>
      <widget class="QWidget" name="graphView" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>471</width>
         <height>321</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="graphPopulate">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>160</y>
         <width>80</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Retrieve</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
